app/src/androidTest/java/de/grobox/transportr/map/MapActivityTest.kt

```diff
@@ -85,6 +86,7 @@ class MapActivityTest : ScreengrabTest() {
     }
 
     @Test
+    @SuppressLint("IgnoreWithoutReason")
     @Ignore // TODO fix
```

Comment:
Maybe the better fix is to give a reason to the `@Ignore`?

Reply:
The problem is that I didn't know the reason, do you remember what exactly didn't work in this test?

Reply:
No, but you can just say currently broken ;)

Reply:
Done.